lib/ansible/modules/cloud/vmware/vmware_host_acceptance.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_host_acceptance
+short_description: Manage acceptance level of ESXi host
+description:
+- This module can be used to manage acceptance level of an ESXi host.
+version_added: '2.5'
+author:
+- Abhijeet Kasurde (@akasurde)
+notes:
+- Tested on vSphere 6.5
+requirements:
+- python >= 2.6
+- PyVmomi
+options:
+  cluster_name:
+    description:
+    - Name of the cluster.
+    - Acceptance level of all ESXi host system in the given cluster will be managed.
+    - If C(esxi_hostname) is not given, this parameter is required.
+  esxi_hostname:
+    description:
+    - ESXi hostname.
+    - Acceptance level of this ESXi host system will be managed.
+    - If C(cluster_name) is not given, this parameter is required.
+  state:
+    description:
+    - Set or list acceptance level of the given ESXi host.
+    - 'If set to C(list), then will return current acceptance level of given host system/s.'
+    - If set to C(present), then will set given acceptance level.
+    - "If set to C(absent), then will set to default value of 'partner'."
+    choices: [ absent, list, present ]
+    required: False
```

Comment:
"state" is required or you should set a default.

Reply:
is "absent" necessary is it's the same as "state = present" + "acceptance_level = partner" ?

Reply:
OK. I will remove absent.